### PR TITLE
Responsive-images-luxeradio-portfolio

### PR DESCRIPTION
**Pull request by cp-priyanka-g**
1. Make all images responsive according to resolution 400w, 800w, 1400w, 2400w.
2. Remove unused CSS from the CTA section
3. Change the Footer image according to Figma.

- [x]  Landing Section
- [x]  Video Section
- [x]  Branding Section
- [x] Element Section
- [x]  Design Section
- [x]  Footer section

**Lighthouse performance:**

Mobile: 98
Desktop:100

### CODE REVIEW

File: vue-frontend/src/luxeradio-data.js [hunk 1]
```diff
@@ -1,28 +1,102 @@
-import landingBackgroundImage from "@/assets/images/portfolio/luxeradio/landing_background.png";
-import videoBackgroundImage from "@/assets/images/portfolio/luxeradio/video_background.png";
+// import landingBackgroundImage from "@/assets/images/portfolio/luxeradio/landing_background.png";
```
Comment (cp-sumi-k) [inline vue-frontend/src/luxeradio-data.js:1]: Remove commented code

**Review by cp-sumi-k — CHANGES_REQUESTED** [review]

**Comment by cp-sumi-k** [comment]
I think you have added some extra `webp` images, which are not used anywhere. You should remove it

File: vue-frontend/src/components/portfolio/LandingSection.vue [hunk 2]
```diff
@@ -10,27 +10,25 @@
     </div>
   </section>
   <section class="background-image tw-relative tw-z-[-1]">
-    <aspect-ratio height="56.26%">
+    <aspect-ratio height="80.26%">
```
Comment (cp-shruti-s) [inline vue-frontend/src/components/portfolio/LandingSection.vue:13]: why are you change this aspect ratio ??

File: vue-frontend/src/components/portfolio/DesignSection.vue [hunk 3]
```diff
@@ -24,18 +24,15 @@
...
       <img
-        v-if="response[0].responsiveImages"
         :src="response[0].backgroundImage[3]"
-        :srcset="`${response[0].backgroundImage[0]} 400w, ${response[0].backgroundImage[1]} 800w, ${response[0].backgroundImage[2]} 1200w, ${response[0].backgroundImage[3]} 1600w`"
-        sizes="(min-width: 992px) 45vw, 100vw"
-        class="tw-w-full tw-h-full tw-object-cover"
+        :srcset="`${response[0].backgroundImage[0]} 400w, ${response[0].backgroundImage[1]} 800w,${response[0].backgroundImage[2]} 1400w,${response[0].backgroundImage[3]} 2400w`"
+        sizes="
+        (min-width: 2420px) 2200px, 
+    (min-width: 720px) calc(94.76vw - 274px),
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/portfolio/DesignSection.vue:31]: Can you explain this calculation?

Comment (cp-priyanka-g) [reply vue-frontend/src/components/portfolio/DesignSection.vue:31]: its width of the image across all three breakpoints.Actually old sizes have rendering problem so i have used this code and its working fine.

File: vue-frontend/src/components/portfolio/BrandingSection.vue [hunk 4]
```diff
@@ -23,19 +23,11 @@
   <section class="background-image tw-relative tw-z-[-1]">
     <aspect-ratio height="56.26%">
       <img
-        v-if="response.responsiveImages"
         :src="response.backgroundImage[3]"
-        :srcset="`${response.backgroundImage[0]} 400w, ${response.backgroundImage[1]} 800w, ${response.backgroundImage[2]} 1200w, ${response.backgroundImage[3]} 1600w`"
-        sizes="(min-width: 992px) 45vw, 100vw"
-        class="tw-w-full tw-h-full tw-object-cover"
+        :srcset="`${response.backgroundImage[0]} 400w, ${response.backgroundImage[1]} 800w, ${response.backgroundImage[2]} 1400w,${response.backgroundImage[3]} 2400w`"
+        sizes="2200px,720px,520px"
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/portfolio/BrandingSection.vue:28]: Remove sizes

**Review by cp-sumi-k — APPROVED** [review]

**Pull request merged**